ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ContractKeysIT.scala

```diff
@@ -177,7 +177,7 @@ final class ContractKeysIT extends LedgerTestSuite {
     } yield {
       assertGrpcError(
         duplicateKeyFailure,
-        Status.Code.INVALID_ARGUMENT,
+        Status.Code.ABORTED,
         Some(Pattern.compile("DuplicateKey|Inconsistent")),
```

Comment:
Can this pattern be refined as well, i.e., which is it?

Reply:
Not sure why I missed it, thanks